src/components/InMoreLanguageSwitch.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div class="wikibase-termbox-subsection-switch">
+		<div class="wikibase-termbox-subsection-switch__switch">
+			<a :href="link" @click="clientShowAllLanguages( $event )">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { VueConstructor } from 'vue';
+import Component, { mixins } from 'vue-class-component';
+import Messages, { MessagesMixin } from './mixins/Messages';
+
+@Component
+export default class TermBox extends ( mixins( Messages ) as VueConstructor<MessagesMixin> ) {
```

Comment:
The class name is not aligned with the file name.

Reply:
c/p error...changed